src/containers/email-form/email-form.jsx

```diff
@@ -1,35 +1,32 @@
 /* eslint-disable no-unused-vars */
-import React, { Component } from 'react';
-import EmailField from '../../components/email-field/email-field';
-import SubmitButton from '../../components/submit-button/SubmitButton';
-import './email-form.css'
+import React, { Component } from "react";
+import EmailField from "../../components/email-field/email-field";
+import SubmitButton from "../../components/submit-button/SubmitButton";
+import "./email-form.css";
 
 class EmailForm extends Component {
-   state = {
-   };
+    state = {};
 
-   continue = e => {
-      e.preventDefault();
-      this.props.nextStep();
-   };
+    continue = e => {
+        e.preventDefault();
+        const { nextStep } = this.props;
 
-   render() {
-      const values = this.props;
```

Comment:
Here what you do is to make a copy of props in values, you want to get the field "values" from props, so you need to add 
"{}" const { values } = this.props